src/cloud_what/providers/azure.py

```diff
@@ -107,12 +107,16 @@ def is_likely_running_on_cloud(self):
         # We know that Azure uses only HyperV
         if 'virt.host_type' in self.hw_info:
             if 'hyperv' in self.hw_info['virt.host_type']:
-                probability += 0.3
+                probability += 0.2
 
         if 'dmi.chassis.asset_tag' in self.hw_info and \
                 self.hw_info['dmi.chassis.asset_tag'] == '7783-7084-3265-9085-8269-3286-77':
             probability += 0.3
 
+        # The waagent log exists and contains the string "Azure Linux Agent"
+        if self._has_azure_linux_agent() == True:
```

Comment:
`E712 comparison to True should be 'if cond is True:' or 'if cond:'`

Reply:
Done. Thanks.